CONTRIBUTING.md

```diff
@@ -83,17 +83,17 @@ We want your work to be readable by others; therefore, we encourage you to follo
 
 #### Types
 
-- Use the strictest types possible for the input, output and object fields. Prefer `Natural`, `Positive` or custom [subrange types](https://nim-lang.org/docs/manual.html#types-subrange-types) to unconstrained `int` where applicable, use `Natural` for indexing.
+- Use the strictest types possible for the input, output, and object fields. Prefer `Natural`, `Positive` or custom [subrange types](https://nim-lang.org/docs/manual.html#types-subrange-types) to unconstrained `int` where applicable, use `Natural` for indexing.
 - On the other hand, write generic code where appropriate. Do not impose arbitrary limitations if the code can work on a wider range of data types.
-- Don't use unsigned numerical types (`uint` and its sized variants), unless wrapping behaviour or binary manipulation is required for the algorithm.
+- Don't use unsigned numerical types (`uint` and its sized variants), unless wrapping behavior or binary manipulation is required for the algorithm.
```

Comment:
This is an international collaboration, so enforcing US spelling in docs is debatable (for symbols it's ok due to tradition).

Reply:
Had I to choose, I would prefer UK spelling and grammar rules. US spelling/grammar feels omnipresent.
As a French speaker, I often mix US/UK usages. Usually, I use Grammarly to help me correct my syntax, but I did not use it for writing the CONTRIBUTING.md file.